packages/xstate-store/src/types.ts

```diff
@@ -109,7 +109,7 @@ export interface Store<
 
 export type AnyStore = Store<any, any, any>;
 
-export type Compute<A extends any> = { [K in keyof A]: A[K] } & unknown;
+export type Compute<A> = { [K in keyof A]: A[K] };
```

Comment:
`& unknown` is required sometimes
```suggestion
export type Compute<A> = { [K in keyof A]: A[K] } & unknown;
```